gamemodes/terrortown/gamemode/shared/sh_playerclass.lua

```diff
@@ -0,0 +1,18 @@
+DEFINE_BASECLASS("player_default")
+
+local PLAYER = {}
+
+PLAYER.WalkSpeed = 220
+PLAYER.RunSpeed = 220
+PLAYER.JumpPower = 160
+PLAYER.CrouchedWalkSpeed = 0.3
```

Comment:
Just a note for the future: It might be neat to let users change those vairables in the UI? Not a change for this PR though

Reply:
Nah i would not want anyone messing with these directly.